ARM-wvd-templates/CreateAndProvisionHostPool/CreateHostpoolTemplate.json

```diff
@@ -592,6 +592,27 @@
         "description": "System data is used for internal purposes, such as support preview features."
       },
       "defaultValue": {}
+    },
+    "securityType": {

```

Comment:
These parameters aren't passed to the virtual machine so it doesn't do anything.